src/trans/gpu/CMakeLists.txt

```diff
@@ -114,6 +114,10 @@ foreach( prec dp sp )
           target_compile_definitions( trans_gpu_${prec} PRIVATE TRANS_SINGLE PARKINDTRANS_SINGLE )
         endif()
 
+        if( ROCM6 AND CMAKE_Fortran_COMPILER_ID MATCHES Cray )
```

Comment:
Is there some missing logic here to set the CMake variable `ROCM6`?

Reply:
The ROCM6 variable is set in the bundle toolchain file for cce 17.0.1. 
Not sure if it's possible to detect that we are using ROCM6.